### PR TITLE
Add support for more Linux distributions

### PR DESCRIPTION
**Pull request by stevapple**
Since Swift 5.2.4 is released for CentOS 8 and Amazon Linux 2 (and also Ubuntu 20.04), here are the patches to Vapor docs for supporting those versions.

### CODE REVIEW

File: 4.0/docs/deploy/digital-ocean.md [hunk 1]
```diff
@@ -4,16 +4,16 @@ This guide will walk you through deploying a simple Hello, world Vapor applicati
...
 ![Create Droplet](../images/digital-ocean-create-droplet.png)
 
-Under distributions, select Ubuntu 18.04 LTS.
+Under distributions, select Ubuntu 18.04 LTS. The following guide will use this version as an example.
 
 ![Ubuntu Distro](../images/digital-ocean-distributions-ubuntu-18.png)
 
 !!! note 
-	You may select any version of Ubuntu that Swift supports. At the time of writing, Swift 5.2 supports 16.04 and 18.04. You can check which operating systems are officially supported on the [Swift Releases](https://swift.org/download/#releases) page.
+	You may select any Linux distribution with a version that Swift supports. At the time of writing, Swift 5.2.4 supports Ubuntu 16.04, 18.04, 20.04, CentOS 8 and Amazon Linux 2. You can check which operating systems are officially supported on the [Swift Releases](https://swift.org/download/#releases) page.
```
Comment (tanner0101) [inline 4.0/docs/deploy/digital-ocean.md:16]: Missing Oxford comma

**Comment by stevapple** [comment]
I'm working on a test version of all-in-one Linux page, see here https://github.com/stevapple/vapor-docs/tree/distro-linux

**Comment by stevapple** [comment]
This also adds descriptions about Fedora, which has been supported long before by community maintainers.

**Comment by stevapple** [comment]
If the new one is okay, I'll merge it into the current branch.

**Comment by 0xTim** [comment]
Agreed, I'd just have one page with the supported platforms and the link to swift.org/download to follow the instructions to install

**Comment by stevapple** [comment]
@0xTim Is the one in stevapple/vapor-docs:distro-linux OK? If it is I would merge it into this PR.

**Comment by 0xTim** [comment]
@stevapple yep looks good to me

File: 4.0/docs/deploy/digital-ocean.md (hunk 1 above)
Comment (stevapple) [reply 4.0/docs/deploy/digital-ocean.md]: Fixed

**Comment by stevapple** [comment]
okay, @0xTim @tanner0101 then we get the all-in-one version in this pr

**Comment by stevapple** [comment]
@0xTim @tanner0101 Would any one of you review the changes?

**Comment by 0xTim** [comment]
Since @tanner0101's original comments about a single `linux.md` file have been addressed I'll merge this anyway

**Pull request merged**